### DISCUSSION TITLE
Custom X axis

### DISCUSSION THREAD

**Discussion by funks81** [RomRider/apexcharts-card#640] [Q&A]
Is there a way of having the x-axis as something other than a datetime value? I'd like to plot average daily temperature on the x-axis vs home energy usage on the y-axis - is there a way to do this?

**Comment by karliwalti** [comment]
The scatter charts would support this: https://apexcharts.com/javascript-chart-demos/scatter-charts/basic/
likely it would be possible to replace the timestamp with some other number in the data generator option https://github.com/RomRider/apexcharts-card?tab=readme-ov-file#data_generator-option
However, I am not sure how values from another entity could be accessed there
